matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/create/CreateLocalRoomTask.kt

```diff
@@ -176,7 +176,7 @@ internal class DefaultCreateLocalRoomTask @Inject constructor(
                 }
 
                 // Give info to crypto module
-                cryptoService.onStateEvent(roomId, event)
+                cryptoService.onStateEvent(roomId, event, null)
```

Comment:
Calling`cryptoService.onStateEvent` here is probably useless, but this PR is not supposed to break anything.